### PR TITLE
Add IHP_SESSION_SECRET placeholder

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -87,6 +87,10 @@
                                 # Uncomment to use a custom database URL
                                 # DATABASE_URL = "postgresql://postgres:...CHANGE-ME";
 
+                                # Uncomment to use a custom session secret, ensuring sessions aren't invalidated
+                                # on each deploy.
+                                # IHP_SESSION_SECRET = "CHANGE-ME";
```
Comment (amitaibu) [inline flake.nix:92]: Should we link to a service like https://it-tools.tech/token-generator , for creating the token?

File: flake.nix [hunk 2]
```diff
@@ -87,6 +87,10 @@
                                 # Uncomment to use a custom database URL
                                 # DATABASE_URL = "postgresql://postgres:...CHANGE-ME";
 
+                                # Uncomment to use a custom session secret, ensuring sessions aren't invalidated
+                                # on each deploy.
```
Comment (mpscholten) [inline flake.nix:91]: We should propably also add a link to the docs for this one https://ihp.digitallyinduced.com/Guide/deployment.html#ihpsessionsecret

File: flake.nix (hunk 1 above)
Comment (mpscholten) [reply flake.nix:92]: I think the session key is a base64 encoded private key, therefore that won't work

Comment (mpscholten) [reply flake.nix:92]: I'll quickly hack a binary that produces a hash on the CLI

File: flake.nix (hunk 2 above)
Comment (amitaibu) [reply flake.nix:91]: Done

**Review by mpscholten — APPROVED** [review]
Thanks :)

**Pull request merged**